packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -63,6 +64,40 @@ def vip_app(container: Container, network: Network, host: str, vip: str):
         raise AssertionError('Unexpected network: {}'.format(network.value))
 
 
+def pod_vip_app(network: Network, host: str, vip: str):
```

Comment:
Please add a doc string here. It would be easy to think that the proxy app and origin app are going to be in the same pod with stating otherwise